.pre-commit-config.yaml

```
@@ -0,0 +1,35 @@
+# See https://pre-commit.com for more information
+# See https://pre-commit.com/hooks.html for more hooks
+repos:
+
+-   repo: https://github.com/pre-commit/pre-commit-hooks
+    rev: v3.2.0
+    hooks:
+    -   id: trailing-whitespace
+        exclude: (migrations/|fixtures/)
+    -   id: end-of-file-fixer
+    -   id: check-yaml
+    -   id: check-added-large-files
+    -   id: no-commit-to-branch
+        args: ["--branch", "prod", "--branch", "dev"]
```

it's not the most ~elegant, but since our current process for deploying to prod is to force-push from the dev branch I think we'll need to relax this.
```suggestion
        args: ["--branch", "dev"]
```